<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TabPropertiesWidget</class>
 <widget class="QWidget" name="TabPropertiesWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="IconSelectButton" name="iconButton">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="tabName">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>&amp;Maximum number of items:</string>
       </property>
       <property name="buddy">
        <cstring>maxItems</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="maxItems">
       <property name="specialValueText">
        <string>default</string>
       </property>
       <property name="maximum">
        <number>100000</number>
       </property>
       <property name="value">
        <number>200</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="storeItems">
     <property name="text">
      <string>&amp;Save Items</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>IconSelectButton</class>
   <extends>QPushButton</extends>
   <header>gui/iconselectbutton.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>iconButton</tabstop>
  <tabstop>maxItems</tabstop>
  <tabstop>storeItems</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
